### PR TITLE
fix staging archive and remove bsc, arbitrum, and optimism testnets

### PR DESCRIPTION
**Pull request by luketchang**

### CODE REVIEW

**Comment by luketchang** [comment]
Passes rust tests

File: archive/staging/22-6-30-deploy/config.json [hunk 1]
```diff
@@ -1159,22 +1260,22 @@
...
-          "optimism-goerli",
+          "optimismgoerli",
           "rinkeby"
         ],
         "domain": 8881,
-        "name": "polygon-mumbai",
+        "name": "polygonmumbai",
         "specs": {
-          "blockExplorer": "https://polygon-mumbai.polygonscan.com/",
+          "blockExplorer": "https://polygonmumbai.polygonscan.com/",
```
Comment (lattejed) [inline archive/staging/22-6-30-deploy/config.json:1274]: Neither of these URLs seem to work

**Review by lattejed — APPROVED** [review]

File: archive/staging/22-6-30-deploy/config.json (hunk 1 above)
Comment (luketchang) [reply archive/staging/22-6-30-deploy/config.json:1306]: Ohh shoot will fix sorry, result of replace all

Comment (luketchang) [reply archive/staging/22-6-30-deploy/config.json:1306]: Nvm not sure what the issue is there, if you could find correct ones and put in that'd be great

**Pull request merged**